### PR TITLE
adding dataprep support for CLIP based models for VideoRAGQnA example for v1.0

### PR DESCRIPTION
**Pull request by srinarayan-srikanthan**
## Description

Add support for CLIP based multimodal dataprep for VideoRAG usecase.

## Issues

The RFC is under review : https://github.com/opea-project/docs/pull/49

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [ ] Others (enhancement, documentation, validation, etc.)

## Dependencies

List the newly introduced 3rd party dependency if exists.

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

**Comment by srinarayan-srikanthan** [comment]
@chensuyue can you please tag this PR with 'V1.0' Milestone

File: comps/dataprep/vdms/README.md
Comment (lvliang-intel) [inline comps/dataprep/vdms/README.md:1]: please add test case for this new component.

File: comps/dataprep/vdms/README.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    -F "files=@./file1.txt" \
+    http://localhost:6007/v1/dataprep
+```
+
+You can specify chunk_size and chunk_size by the following commands.
+
+```bash
+curl -X POST \
+    -H "Content-Type: multipart/form-data" \
+    -F "files=@/home/sdp/yuxiang/opea_intent/GenAIComps4/comps/table_extraction/LLAMA2_page6.pdf" \
```
Comment (XinyuYe-Intel) [inline comps/dataprep/vdms/README.md:144]:
```suggestion
    -F "files=@/home/sdp/yuxiang/opea_intent/GenAIComps4/comps/table_extraction/LLAMA2_page6.pdf" \
```
    -F "files=@./LLAMA2_page6.pdf

**Comment by BaoHuiling** [comment]
@chensuyue  hello Suyue, the File change warning outputs below:
+ find /home/runner/work/GenAIComps/GenAIComps/.github/workflows/docker/compose/ -name '*dataprep*'
+ grep -q .
+ '[' -n 'comps/dataprep/vdms/langchain/docker/Dockerfile
comps/dataprep/vdms/multimodal_langchain/docker/Dockerfile' ']'
+ grep -q dataprep-compose-cd.yaml
+ echo 'Please check if the added comps/dataprep/vdms/multimodal_langchain/docker/Dockerfile is included in the yaml under path /home/runner/work/GenAIComps/GenAIComps/.github/workflows/docker/compose/dataprep-compose-cd.yaml.'
Please check if the added comps/dataprep/vdms/multimodal_langchain/docker/Dockerfile is included in the yaml under path /home/runner/work/GenAIComps/GenAIComps/.github/workflows/docker/compose/dataprep-compose-cd.yaml.
+ no_add=TRUE
+ [[ TRUE == \T\R\U\E ]]
+ exit 1
Error: Process completed with exit code 1.

Do we need to add this PR into the /workflows/docker/compose/dataprep-compose-cd.yaml file? It seems not all dataprep microsevices are in it.

File: comps/dataprep/vdms/README.md
Comment (srinarayan-srikanthan) [reply comps/dataprep/vdms/README.md:1]: added

**Comment by BaoHuiling** [comment]
@lvliang-intel  @Spycsh @XinyuYe-Intel We see the repo is been reorged, pls, help review this PR so we could be merged today

**Review by yinghu5 — APPROVED** [review]

File: .github/workflows/docker/compose/dataprep-compose-cd.yaml [hunk 2]
```diff
@@ -23,3 +23,6 @@ services:
     build:
       dockerfile: comps/dataprep/pinecone/langchain/Dockerfile
     image: ${REGISTRY:-opea}/dataprep-pinecone:${TAG:-latest}
+  dataprep-vdms:
+    build:
+      dockerfile: comps/dataprep/vdms/multimodal_langchain/docker/Dockerfile
```
Comment (chensuyue) [inline .github/workflows/docker/compose/dataprep-compose-cd.yaml:28]: Need a image name for the new image.

**Comment by chensuyue** [comment]
> @chensuyue hello Suyue, the File change warning outputs below:
> 
> * echo 'Please check if the added comps/dataprep/vdms/multimodal_langchain/docker/Dockerfile is included in the yaml under path /home/runner/work/GenAIComps/GenAIComps/.github/workflows/docker/compose/dataprep-compose-cd.yaml.'
>   Please check if the added comps/dataprep/vdms/multimodal_langchain/docker/Dockerfile is included in the yaml under path /home/runner/work/GenAIComps/GenAIComps/.github/workflows/docker/compose/dataprep-compose-cd.yaml.
> * no_add=TRUE
> * [[ TRUE == \T\R\U\E ]]
> * exit 1
>   Error: Process completed with exit code 1.
> 
> I have added VDMS into the /workflows/docker/compose/dataprep-compose-cd.yaml file, but what is this used for?

This is new for release image build/scan/publish.

File: .github/workflows/docker/compose/dataprep-compose-cd.yaml (hunk 2 above)
Comment (BaoHuiling) [reply .github/workflows/docker/compose/dataprep-compose-cd.yaml:28]: Name added. Thanks for reminder

**Review by lvliang-intel — APPROVED** [review]

**Pull request merged**